frontend/src/components/BackLink/index.module.scss

```diff
@@ -1,5 +1,13 @@
 .backLinkWrapper {
-  margin: 24px 0 92px 24px;
+  margin: 24px 0 24px 24px;
+}
+
+button {
```

Comment:
Did the linter not complain about using `button` in the wild? Without converting it to a class?

Reply:
The property order is also whack according to linter styles. I think your linter isn't working?

Reply:
I can run this branch through my VSCode tomorrow and maybe it will fix this issue